src/runtime/schema/settings.ts

```diff
@@ -132,6 +132,8 @@ function withAutoTypegenConfig(
   )
   nexusConfig.typegenAutoConfig = undefined
 
+  console.log({ typegen: typegenAutoConfigObject.sources })
```

Comment:
We'll remove once done debugging

Reply:
Removed it ayway, I feel bad pushing a `console.log` on master 🙈